servers/rendering/shader_compiler.cpp

```diff
@@ -1220,7 +1221,7 @@ String ShaderCompiler::_dump_node_code(const SL::Node *p_node, int p_level, Gene
 						}
 
 						String node_code = _dump_node_code(onode->arguments[i], p_level, r_gen_code, p_actions, p_default_actions, p_assigning);
-						if (!RS::get_singleton()->is_low_end() && is_texture_func && i == 1) {
```

Comment:
This is what is causing the issue on the OpenGL renderer. Removing this low end check makes it so that the shader compiler replaces the sampler name with ``sampler2D(sampler_name, sampler[])`` which we explicitly do not want to do in the openGL renderer as we use combined texture samplers there. 

For low end we need to add another condition to set ``multiview_check_needed``. I would undo this change here and then restructure the branch as follows:
```
if (!RS::get_singleton()->is_low_end() && is_texture_func && i == 1) {
	...
} else if (is_texture_func && i == 1) {
	multiview_check_needed = ... // detect if multiview_check_needed
	code += node_code;
} else if (multiview_check_needed && i == 2) {
	// UV coordinate after using color, depth or normal roughness texture.
	code += "multiview_uv(" + node_code + ".xy)";
} else {
	code += node_code;
}
```

Reply:
Ah, that makes sense, it was getting confused with the merge on those lines and thought you changed it in another direction. ok will fix that up

Reply:
I'm double checking `is_low_end` as it has little cost and properly writes out the scenario where we're checking for multiview for OpenGl, this in case someone ever changes or adds to the ifs checked and breaks the system :)